### ISSUE TITLE
Ensure that the run completed message contains associated person data

### ISSUE THREAD

**Issue by bryankennedy** [cns-iu/xmacroscope#144]
CNS' MAV code relies upon the run completed graphql message subscription. Ensure that the person data that is linked to the run via an association is passed along in this message.

**Comment by bryankennedy** [comment]
@bherr2 I am publishing `race-initiated` when the runStart resolver runs. Could you look through the returned data and confirm that it generates what MAV is looking for?

Relevant code: https://github.com/cns-iu/xmacroscope/blob/smm-sprint-2-mav-integration/packages/serverGraphQL/src/graphql/resolvers/run.js#L38-L50

I tried running MAV alongside the server, but I don't think I have it correctly looking at the server on my local copy.

I'm working on the `run-selected` and `race-completed` messages, but I just wanted to confirm that this is what you're looking for.

**Comment by bryankennedy** [comment]
@bherr2 could you explain how I point the MAV to look at the server instead of your mocking code? I am watching the raw data screen on the MAV app, but it's obviously just looking at the local mocker and not my server subscriptions. Just want to make sure I'm not missing an obvious configuration step.

**Comment by bherr2** [comment]
`yarn ng serve --env=local` from aisl package

**Comment by bryankennedy** [comment]
Thanks, I'll add something to the documentation about how to run that. I'll post back when I can confirm that the subscription messages are coming through.

**Comment by bryankennedy** [comment]
I'm getting a config error, when I run that:
```
Cannot read property 'config' of null
TypeError: Cannot read property 'config' of null
    at Class.run (/Users/bkennedy/Desktop/Data Visualization NSF/src/xmacroscope/node_modules/@angular/cli/tasks/serve.js:51:63)
    at check_port_1.checkPort.then.port (/Users/bkennedy/Desktop/Data Visualization NSF/src/xmacroscope/node_modules/@angular/cli/commands/serve.js:123:26)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```
Is there a default config file that I can copy and setup? I suspect that local env is looking for a `.env` file but I'm not sure what elements I should configure.

**Comment by bherr2** [comment]
I can make a package.json script to make it easier. The environments dir is in aisl/src/environments/environment.local.ts which should connect to the local smm server.

**Comment by bherr2** [comment]
Just pushed to develop

**Comment by bherr2** [comment]
wrt to `race-initiated` that looks right. You can see the model we are expecting here: 
* https://github.com/cns-iu/xmacroscope/blob/develop/packages/aisl-api/src/models/aisl-messages.ts
* https://github.com/cns-iu/xmacroscope/tree/develop/packages/aisl-api/src/models

**Comment by bryankennedy** [comment]
@bherr2 I'm making progress on this issue. I ran into a a bug with my association code that I had to sort out, but should be able to commit a fix for the entire set of messages that you need, later today.

**Comment by bryankennedy** [comment]
This is resolved with #148 being merged into develop.
